prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml

```diff
@@ -16,7 +16,7 @@ job_template: &job_template
         command:
```

Comment:
What about job name, now is `pre-master-kyma-tools-etcd-backup` maybe should be `pre-master-kyma-tools-etcd-backup-job`?

Reply:
i changed both jobs names:
`pre-master-kyma-tools-etcd-backup` --> `pre-master-kyma-components-etcd-backup-job`
`pre-master-kyma-tools-etcd-tls-setup` --> `pre-master-kyma-components-etcd-tls-setup-job`